src/Storefront/Controller/AuthController.php

```diff
@@ -312,4 +317,42 @@ public function resetPassword(RequestDataBag $data, SalesChannelContext $context
 
         return $this->redirectToRoute('frontend.account.profile.page');
     }
+
+    #[Route(path: '/account/login/customer/{token}/{customerId}/{userId}', name: 'frontend.account.login.customer', methods: ['GET'])]
```

Comment:
Is it somehow possible that we do a POST from the Administration? Writing to Session is should be avoided in a "read" operation aka GET

Reply:
Do you mean to just change it to "POST"?

Reply:
I don't know, if this works with "window.open" from the administration.

Reply:
Changed it to use "POST" in PHP and perform the post method from the administration in https://github.com/shopware/shopware/pull/3727/commits/a104ab893190bb0bc26cfbc658bad52a62b34ae7